docs/guide/hot-reload.md

```diff
@@ -4,6 +4,8 @@ Vuex supports hot-reloading mutations, modules, actions and getters during devel
 
 For mutations and modules, you need to use the `store.hotUpdate()` API method:
 
+## Example
```

Comment:
I think we can remove this heading since the previous text is directly referencing the code below.

Having "Dynamic module hot reloading" on the next section is good 👍

Reply:
Fixed in f096b72.